Project.toml

```diff
@@ -1,17 +1,20 @@
 name = "MockTableGenerators"
 uuid = "63c704a1-a924-4685-a9dd-2c8225e4c0d3"
 authors = ["Beacon Biosignals Inc."]
-version = "0.1.0"
+version = "0.2.0"
 
 [deps]
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
```

Comment:
I don't think we need to include `Random` in these changes?

Reply:
yeah it's defined in `Random`

Reply:
Ah.. missed that! :+1: Just saw `rand` which is `Base`